deploy/cluster_role.yaml

```diff
@@ -80,3 +81,17 @@ rules:
       - securitycontextconstraints
     verbs:
       - '*'
+  - apiGroups:
+      - keda.sh
+    resources:
+      - '*'
```

Comment:
Let's avoid using wildcards where we can. we already have a BZ regarding it (https://bugzilla.redhat.com/show_bug.cgi?id=2052928)